### ISSUE TITLE
Сheerpj runtime pollutes global context and interferes with another frameworks

### ISSUE THREAD

**Issue by sergeypayu** [leaningtech/cheerpj-meta#60]
I cannot longer use cheerpj after upgrading to 1.3. The reason is that your compiler named one of the functions inside icu.js as 'ga'. This function is placed in global context and it interferes with google analytics code, which defines the same function by default (https://developers.google.com/analytics/devguides/collection/analyticsjs/)

Now when I call ga('send', 'event', 'title', 'subtitle'), your function is called instead of google's, and I get the following error:

```
TypeError: Cannot read property 'i3' of undefined
    at La (https://cjrt.leaningtech.com/1.3/icu.js:1:5131404)
    at ga (https://cjrt.leaningtech.com/1.3/icu.js:1:1327222)
```

This is a part of a bigger problem of polluting the global context. After initializing cheerpj A LOT of stuff is added to the window object, new properties are added to almost every object. For example Uint8Array obtains a prototype property 'ifs'. Is this intentional and is there a way to avoid these problems and place all cheerpj related stuff to a specific context?

**Comment by alexp-sssup** [comment]
CheerpJ needs to have properties in both the global space and global objects for either performance or functional reasons (depending on the specific cases). We are aware that this could be a problem and we plan to improve the situation eventually.

Since the specific problem you are facing is related to Google Analytics, which is heavily used, we have now reserved the ```ga``` symbol. The fix will be included in CheerpJ 1.4, but you can test now by using out "latest" endpoint: ```https://cjrtnc.leaningtech.com/latest/loader.js```

**Comment by sergeypayu** [comment]
Understood. Thanks for the quick fix!
